py-polars/tests/unit/interchange/test_roundtrip.py

```diff
@@ -28,27 +28,6 @@
 ]
 
 
-@given(dataframes(allowed_dtypes=protocol_dtypes, excluded_dtypes=[pl.Boolean]))
-def test_roundtrip_polars_parametric(df: pl.DataFrame) -> None:
-    dfi = df.__dataframe__()
-    with pl.StringCache():
-        result = pl.from_dataframe(dfi)
-    assert_frame_equal(result, df, categorical_as_str=True)
-
-
-@given(
-    dataframes(
-        allowed_dtypes=protocol_dtypes,
-        excluded_dtypes=[pl.Boolean, pl.Categorical],
-        chunked=False,
-    )
-)
-def test_roundtrip_polars_zero_copy_parametric(df: pl.DataFrame) -> None:
-    dfi = df.__dataframe__(allow_copy=False)
-    result = pl.from_dataframe(dfi, allow_copy=False)
-    assert_frame_equal(result, df, categorical_as_str=True)
```

Comment:
These tests don't make much sense anymore, as the fast path makes sure these do not end up going through the interchange protocol at all.